proto/servicemanager/v4/servicemanager.proto

```diff
@@ -0,0 +1,334 @@
+}
+
+message GetNodeMonitoring {}
+
+enum ConnectionEnum {
+    DISCONNECTED = 0;
+    CONNECTED    = 1;
+}
+
+message ConnectionStatus {
+    ConnectionEnum cloud_status = 1;
+}
+
+message ImageContentInfo {
+    uint64              request_id  = 1;
+    repeated ImageFile  image_files = 2;
+    common.v1.ErrorInfo error       = 3;
+}
+
+message ImageFile {
+    string relative_path = 1;
+    bytes  sha256        = 2;
+    uint64 size          = 3;
+}
+
+message ImageContent {
+    uint64 request_id    = 1;
+    string relative_path = 2;
+    uint64 parts_count   = 3;
+    uint64 part          = 4;
+    bytes  data          = 5;
+}
+
+message SMOutgoingMessages {
+    oneof SMOutgoingMessage {
+        UnitConfigStatus      unit_config_status      = 1;
+        RunInstancesStatus    run_instances_status    = 2;
+        UpdateInstancesStatus update_instances_status = 3;
+        OverrideEnvVarStatus  override_env_var_status = 4;
+        LogData               log                     = 5;
+        NodeMonitoring        node_monitoring         = 6;
+        Alert                 alert                   = 7;
+        ImageContentRequest   image_content_request   = 8;
+        ClockSyncRequest      clock_sync_request      = 9;
+    }
+}
+
+message Partition {
+    string          name       = 1;
+    repeated string types      = 2;
+    uint64          total_size = 3;
+}
+
+message UnitConfigStatus {
+    string              vendor_version = 1;
+    common.v1.ErrorInfo error          = 2;
+}
+
+message RunInstancesStatus {
+    repeated InstanceStatus instances = 1;
+}
+
+message UpdateInstancesStatus {
+    repeated InstanceStatus instances = 1;
+}
+
+message InstanceStatus {
+    InstanceIdent       instance    = 1;
+    uint64              aos_version = 2;
+    string              run_state   = 3;
+    common.v1.ErrorInfo error_info  = 4;
+}
+
+message InstanceIdent {
+    string service_id = 1;
+    string subject_id = 2;
+    int64  instance   = 3;
+}
+
+message OverrideEnvVarStatus {
+    repeated EnvVarInstanceStatus env_vars_status = 1;
+    common.v1.ErrorInfo           error           = 2;
+}
+
+message EnvVarInstanceStatus {
+    InstanceIdent         instance    = 1;
+    repeated EnvVarStatus vars_status = 2;
+}
+
+message EnvVarStatus {
+    string              var_id = 1;
+    common.v1.ErrorInfo error  = 2;
+}
+
+message LogData {
+    string              log_id     = 1;
+    uint64              part_count = 2;
+    uint64              part       = 3;
+    bytes               data       = 4;
+    common.v1.ErrorInfo error      = 5;
+}
+
+message NodeMonitoring {
+    google.protobuf.Timestamp   timestamp           = 1;
+    MonitoringData              monitoring_data     = 2;
+    repeated InstanceMonitoring instance_monitoring = 3;
+}
+
+message MonitoringData {
+    uint64                  ram         = 1;
+    uint64                  cpu         = 2;
+    repeated PartitionUsage disk        = 3;
```

Comment:
disks to be uniform with other repeated fields

Reply:
done